modules/src/ics04_channel/msgs/recv_packet.rs

```diff
@@ -18,22 +18,23 @@ pub const TYPE_URL: &str = "/ibc.core.channel.v1.MsgRecvPacket";
 #[derive(Clone, Debug, PartialEq)]
 pub struct MsgRecvPacket {
     pub packet: Packet,
-    proofs: Proofs,
-    signer: AccountId,
+    pub proofs: Proofs,
```

Comment:
@cezarad FYI the diff hunks here. As soon as we merge the present PR, your PR on recv_packet (https://github.com/informalsystems/ibc-rs/pull/737) will need to be adapted.